### ISSUE TITLE
Data base info items

### ISSUE THREAD

**Issue by krozs** [ztizzlegaming/CSGOWinBig#23]
How do you add items them to the database? 
Do you use an website to add info to the database?

Thanks,

**Comment by joka503** [comment]
We also want to do a history page where users can see the previous winners and the entire pot as seen in the printscreen: http://prntscr.com/8llnpw
But in the winner section we want it to present us with the avatar from the Steam profile as well the link to it, just as like the index page when someone makes a bet.
All of this will be inside of a table but the trouble is since the code is inside the script i have no idea how the html page calls the functions from the script file.
So basically we are using the data from the history table from the database and expose it in a table in a html page but with the avatar like the profile page as this: http://prntscr.com/8llpan

**Comment by TheAnthonyNL** [comment]
@joka503 
that history is coming soon from me.
http://timekillerz.eu/skins/history.php

**Comment by alanfort** [comment]
@TheAnthonyNL  can u share please?

**Comment by TheAnthonyNL** [comment]
Soon i will lol...
